src/sections/Community/Handbook/connect.js

```diff
@@ -200,8 +201,8 @@ const Connect = () => {
             </p>
             <div className="channels-list">
               <p className="channels-para">
-                <img
-                  className="channels-img"
+                <StaticImage
+                  className="channels-StaticImage"
```

Comment:
Update the same in the handbook.style.js file too

Reply:
done :+1: